lib/rucio/core/rse.py

```diff
@@ -725,7 +735,7 @@ def get_rse_supported_checksums(rse_id, session=None):
     """
     Retrieve from the DB and parse the RSE attribute defining the checksum supported by the RSE
     """
-    return parse_checksum_support_attribute(get_rse_attribute(key=CHECKSUM_KEY, rse_id=rse_id, session=session))
+    return parse_checksum_support_attribute([get_rse_attribute(rse_id, CHECKSUM_KEY, session=session)])
```

Comment:
Conflicts with https://github.com/rucio/rucio/pull/5669, but I can rebase that afterwards and fix this weird list-wrapping.

Reply:
Let's handle conflicts once they arrive. The one who gets merged first has priority.